### PR TITLE
chore: merge passkeys/webauthn docs

### PR DESCRIPTION
**Pull request by vinckr**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/kratos/passwordless/05_passkeys.mdx [hunk 1]
```diff
@@ -154,3 +168,159 @@ passkey found on the device will be used to sign in.
 
 </BrowserWindow>
 ```
+
+## WebAuthn
+
+:::note
+
+Please use the PassKey method instead. This is documented for legacy reasons.
```
Comment (aeneasr) [inline docs/kratos/passwordless/05_passkeys.mdx:176]: This isn't true, WebAuthn Passwordless is deprecated. WebAuthn is still a 2FA method for hardware keys.

Comment (aeneasr) [reply docs/kratos/passwordless/05_passkeys.mdx:176]: My bad, this is the passwordless section. It wasn't clear to me, in that case this is correct

**Review by aeneasr — APPROVED** [review]

**Comment by aeneasr** [comment]
Merge conflict

File: docs/kratos/passwordless/05_passkeys.mdx (hunk 1 above)
Comment (vinckr) [reply docs/kratos/passwordless/05_passkeys.mdx:176]: We can also explicitly mention that it is legacy only for first-factor but you can still use it for second factor 🤔

Comment (aeneasr) [reply docs/kratos/passwordless/05_passkeys.mdx:176]: Sure!

**Pull request merged**